client/src/app/components/hoc/TabbedSection.tsx

```diff
@@ -1,15 +1,15 @@
 import classNames from "classnames";
-import React, { useState } from "react";
+import React, { useEffect, useState } from "react";
 import Modal from "../Modal";
 import { ModalData } from "../ModalProps";
 import Spinner from "../Spinner";
 import "./TabbedSection.scss";
 
 interface TabOption {
     /**
-     * Is the Tab disabled (it will make the tab unclickable).
+     * Is the Tab enabled (if not enabled the tab will be unclickable).
      */
-    disabled?: boolean;
+    enabled?: boolean;
```

Comment:
What's the reason for this switch ? Why should we have to pass a boolean to have the tabs "enabled", instead of that being the default ? 
You also had to add a useEffect to support it.

Reply:
I added UseEffect so the first available enabled tab gets selected, previously it was hardcoded to tab 0. I also changed the working from disabled to enabled because naming convention for booleans should be "positive" so we avoid dobule negation !sDisabled